pkg/textutil/util.go

```diff
@@ -0,0 +1,17 @@
+package textutil
+
+import (
+	"strings"
+	"unicode"
+)
+
+func StripSpaces(str string) string {
```

Comment:
can you use strings.ReplaceAll instead?

https://pkg.go.dev/strings#ReplaceAll

Reply:
because i wanna strip all white space including `\t`, `\n`, etc.. and i could find a way to convert `unicode.Space` to a string/[]rune

Reply:
ok

Reply:
maybe it's not really necessary, i'll test some things out